datumaro/util/os_util.py

```diff
@@ -49,6 +66,57 @@ def walk(path, max_depth=None):
 
         yield dirpath, dirnames, filenames
 
+def copytree(src, dst):
+    # Shutil works very slow pre 3.8
+    # https://docs.python.org/3/library/shutil.html#platform-dependent-efficient-copy-operations
+    # https://bugs.python.org/issue33671
+
+    if sys.version_info[1] >= 8:
+        shutil.copytree(src, dst)
+        return
+
+    basedir = osp.dirname(dst)
+    if basedir:
+        os.makedirs(basedir, exist_ok=True)
+
+    if sys.platform == 'windows':
+        # Ignore
+        #   B603: subprocess_without_shell_equals_true
+        #   B607: start_process_with_partial_path
+        # In this case we control what is called and command arguments
+        # PATH overriding is considered low risk
+        subprocess.check_call(["xcopy", src, dst, "/s", "/e"]) # nosec
```

Comment:
`xcopy` seems to be very finicky with its parameters; for example, I tried this:

```console
> xcopy .\venv\ test-dir\ /s /e
Invalid path
0 File(s) copied
```

It also outputs progress to stdout and errors to stderr, so you need to capture its output. Frankly, I don't know if dealing with it is worth the hassle.

Reply:
Also, `xcopy`'s second parameter is optional, so if `dst` looks like an option, then the command might get interpreted in an unexpected way (and I don't know if there's a `--` equivalent to prevent this).

Reply:
I added few more parameters to the call and added the command output on error. Unless there is a more performant solution, it's better to be used.